hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/TestRMWebServicesNodeLabels.java

```diff
@@ -69,13 +73,22 @@
 
   private static final Logger LOG = LoggerFactory
       .getLogger(TestRMWebServicesNodeLabels.class);
+  private static final String NODE_0 = "nid:0";
+  private static final String NODE_1 = "nid1:0";
+  private static final String NODE_2 = "nid2:0";
+  private static final String LABEL_A = "a";
+  private static final String LABEL_B = "b";
+  private static final String LABEL_X = "x";
+  private static final String LABEL_Y = "y";
+  private static final String LABEL_Z = "z";
 
   private static MockRM rm;
   private static YarnConfiguration conf;
 
   private static String userName;
   private static String notUserName;
   private static RMWebServices rmWebService;
+  public static final boolean DEFAULT_NL_EXCLUSIVITY = true;
```

Comment:
Nit: This could be better positioned after all the static final declarations.

Reply:
Fixed.